src/Command/SonataDumpDoctrineMetaCommand.php

```diff
@@ -21,6 +21,8 @@
 
 /**
  * Return useful data on the database schema.
+ *
+ * @deprecated translator property is deprecated since version 3.x, to be removed in 4.0
```

Comment:
What is this translator property? :confused:

Reply:
copy & paste 😆